tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/common/ExecuteTest.java

```diff
@@ -67,6 +71,8 @@
         optionSettings.put("Map", Map.class);
         optionSettings.put("Add WMS", WmsSettings.class);
         optionSettings.put("Remove WMS", RemoveWmsSettings.class);
+        optionSettings.put("Add WMTS", WmtsSettings.class);
+        optionSettings.put("Remove WMTS", RemoveWmtsSettings.class);
```

Comment:
I thought we were going to consolidate Add/Remove into one and create a dialog for selection as part of opetionItem menu cleanup?